core/src/main/java/kafka/server/SharePartitionManager.java

```diff
@@ -694,11 +695,11 @@ public ShareGroupMetrics(Metrics metrics, Time time) {
                 metrics.metricName(
                     SHARE_ACK_RATE,
                     METRICS_GROUP_NAME,
-                    "The total number of offsets acknowledged for share groups per second."),
+                    "The total number of requests acknowledged for share groups per second."),
```

Comment:
I'd prefer "The rate of acknowledge requests." We know it's share groups because of the group.